### PR TITLE
Climate reality project

### PR DESCRIPTION
**Pull request by yeah800**
first infos

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -739,4 +739,48 @@ exports.data = [
...
         },
     },
+    {
+        meta: {
+            name: "Climate Reality Project",
+            link: "https://www.climaterealityproject.org/",
+            label: "climate-realtiy-project",
+        },
+        description: {
+            content: "Founded by former Vice President Al Gore Climate Reality Project tries to reach their vision of a global network of climate activists.",
```
Comment (reasn) [inline data.js]: Suggest `Founded by former Vice President Al Gore, the Climate Reality works to establish a global network of climate activists by regular trainings and supporting them in spreading the word.`

File: data.js [hunk 2]
```diff
@@ -739,4 +739,48 @@ exports.data = [
...
+            updatedBy: "Andre", updatedAt: "2020-01-20"
+        },
+        established: {
+            content: "(2006)[https://www.climaterealityproject.org/whoweare]",
+            updatedBy: "Andre", updatedAt: "2020-01-20"
+        },   
+        kpis: {
+                content: "- presentations / pa \n- new climate reality leaders / pa \n- downloaded resources / pa",
```
Comment (reasn) [inline data.js]: Suggest `- presentations\n- newly trained climate reality leaders (people able to hold talks and spread the word)\n`

File: data.js [hunk 3]
```diff
@@ -739,4 +739,48 @@ exports.data = [
...
+                updatedBy: "Andre", updatedAt: "2020-01-20",
+        },
+        actions: {
+            content: "- holding presentations in 150 countries; individuals can (request)[https://realityhub.climaterealityproject.org/events/attendpresentation?utm_source=24hoursofreality.org&utm_medium=referral&utm_campaign=24Hours&segment=web_homepage_tile&_ga=2.238082944.1453822528.1579548432-1993394381.1579548432] a speaker \n- holding up team meetings around the US \n- and other (initiatives)[https://climaterealityproject.org/initiative] ",
```
Comment (reasn) [inline data.js]: 1. UTM params are unecessary, link -> `https://realityhub.climaterealityproject.org/events/attendpresentation`
Can you add examples for "other"? `\n- other initiatives (example 1, example 2, ...)`

File: data.js [hunk 4]
```diff
@@ -739,4 +739,48 @@ exports.data = [
...
+            updatedBy: "Andre", updatedAt: "2020-01-20",
+        
+        activeCountries: {
+                content: "(Branches)[https://climaterealityproject.org/internationalbranches] exist all over the world",
```
Comment (reasn) [inline data.js]: Number of countries or some more detail would be great

File: data.js [hunk 5]
```diff
@@ -739,4 +739,48 @@ exports.data = [
...
+        yearlyIncome: {
+            content: "($ 26M)[https://www.climaterealityproject.org/2018-annual-report?_ga=2.125057674.1453822528.1579548432-1993394381.1579548432#arfinance] in 2018",
+            updatedBy: "Andre", updatedAt: "2020-01-20"
+        },
+        employees: {
+            content: "(83)[https://climaterealityproject.org/whoweare]",
+            updatedBy: "Andre", updatedAt: "2020-01-20"
+        },
+        participation: {
+            content: "(20,000 People)[https://www.climaterealityproject.org/our-mission] presenting (so called (Climate Reality Leaders)[https://climaterealityproject.org/climatespeakers?_ga=2.129118732.1453822528.1579548432-1993394381.1579548432]) on the topic of climate crisis & (100)[https://www.climaterealityproject.org/our-mission] teams meeting (so called (chaperts)[https://www.climaterealityproject.org/chapters])",
```
Comment (reasn) [inline data.js]: To be consistent with the wording above, let's say `"(20,000 People Climate Reality Leaders)[https://www.climaterealityproject.org/our-mission] who present and spread the word`

File: data.js [hunk 6]
```diff
@@ -739,4 +739,48 @@ exports.data = [
...
+            updatedBy: "Andre", updatedAt: "2020-01-20"
+        },
+        pressArticles: {
```
Comment (reasn) [inline data.js]: Empty, imo redundant

**Review by reasn — CHANGES_REQUESTED** [review]

File: data.js (hunk 6 above)
Comment (yeah800) [reply data.js]: done

File: data.js (hunk 5 above)
Comment (yeah800) [reply data.js]: changed little differently. The second link was important too

File: data.js (hunk 4 above)
Comment (yeah800) [reply data.js]: changed the wording

File: data.js (hunk 1 above)
Comment (yeah800) [reply data.js]: suggestion accepted

File: data.js (hunk 2 above)
Comment (yeah800) [reply data.js]: changed to wording

File: data.js (hunk 3 above)
Comment (yeah800) [reply data.js]: done

**Comment by yeah800** [comment]
@reasn please have another look

**Review by reasn — APPROVED** [review]
LGTM

File: data.js [hunk 7]
```diff
@@ -739,4 +739,44 @@ exports.data = [
...
+        established: {
+            content: "(2006)[https://www.climaterealityproject.org/whoweare]",
+            updatedBy: "Andre", updatedAt: "2020-01-20"
+        },   
+        kpis: {
+                content: "- presentations / pa \n- newly trained climate reality leaders (people able to hold talks and spread the word) / pa \n- downloaded resources / pa",
+                updatedBy: "Andre", updatedAt: "2020-01-20",
+        },
+        actions: {
+            content: "- holding presentations in 150 countries; individuals can (request)[https://realityhub.climaterealityproject.org/events/attendpresentation] a speaker \n- holding up team meetings around the US \n- and other (initiatives)[https://climaterealityproject.org/initiative] like a collaboration with the musician Jack Johnson",
```
Comment (reasn) [inline data.js:761]: @yeah800 I didn't double-check thoroughly, sorry. Most hyperlinks in this PR are wrong. Please fix

**Pull request merged**